11_install_agent/80_agent_configuration.txt

```diff
@@ -16,8 +15,12 @@ EOF
 ====
 
 We advise you to use the +IP address+ of the Rudder Root Server. The DNS name of
-this server can also be accepted if you have a complete DNS infrastructure
-matching the IP of the Nodes with their hostnames.
+this server can also be accepted if you have a trusted DNS infrastructure
+with proper reverse resolutions.
+
+Also, by default, Rudder will try to contact a server named "rudder" if this file
```

Comment:
It has been reverted for 3.0 because there was really unexpected behaviour, especially in env with several rudder srever (i.e a prod where the prod env IS rudder.company.com, and dev is rudder-dev.company.com - by default, new nodes, especially testing one, will go to prod. We need to find a better way to handle that).